src/bundle/Core/Command/DebugConfigResolverCommand.php

```diff
@@ -95,6 +107,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $scope = $input->getOption('scope');
         $parameterData = $this->configResolver->getParameter($parameter, $namespace, $scope);
 
+        if (null !== ($sort = $input->getOption('sort')) && is_array($parameterData) && is_array($parameterData[0]) && array_key_exists($sort, $parameterData[0]) && is_scalar($parameterData[0][$sort])) {
+            if ($input->getOption('reverse-sort')) {
+                usort($parameterData, static function ($a, $b) use ($sort) {
+                    return $b[$sort] <=> $a[$sort];
+                });
+            } else {
+                usort($parameterData, static function ($a, $b) use ($sort) {
```

Comment:
```suggestion
                usort($parameterData, static function ($a, $b): int use ($sort) {
```

Could you also please specify types for `$a` and `$b` ?